### PR TITLE
New RHAI pages

### PR DESCRIPTION
**Pull request by quarckster**
Added model pages for RHAI Rules, Planner, Actions and Help. Extended `test_navigation` for all pages in RHAI.

### CODE REVIEW

**Comment by quarckster** [comment]
@oshtaier @abalakh please review

File: robottelo/ui/locators/menu.py [hunk 1]
```diff
@@ -219,7 +219,7 @@
     "insights.rules": (
         By.XPATH,
         (MENU_CONTAINER_PATH +
-         "//a[@href='/redhat_access/insights/rules/']")),
+         "//a[@href='/redhat_access/insights/rules']")),
```
[the remark this answers is not in the document]
Comment (quarckster) [reply robottelo/ui/locators/menu.py]: @abalakh it's just a fix, I couldn't navigate to that page without removing `/`

[the remark this answers is not in the document]
Comment (abalakh) [reply robottelo/ui/locators/menu.py]: sorry, wrong line :) i was rather asking about new links like `insights.actions`, `insights.planner` and `insights.help`

[the remark this answers is not in the document]
Comment (quarckster) [reply robottelo/ui/locators/menu.py]: I don't think it's needed. `href` value is unique and besides I didn't invent nothing new here, I just followed the previous examples.

[the remark this answers is not in the document]
Comment (abalakh) [reply robottelo/ui/locators/menu.py]: sure, treat it as optional nitpick. `href` is unique but may change in future release, while id is usually not touched. Regarding previous examples - all the rest links except insights are using ids, that's why i asked.

File: robottelo/ui/locators/base.py [hunk 2]
```diff
@@ -2631,6 +2635,22 @@
...
+        By.XPATH,
+        "//span[normalize-space(.)='Issues']/../div[@class='num']"
+    ),
+    "insights.help.general_info": (
+        By.XPATH,
+        "//div[@ng-app='RedhatAccessInsights']//h3[1]"
+    ),
+    "insights.help.getting_started": (
+        By.XPATH,
+        "//div[@ng-app='RedhatAccessInsights']//h3[2]"
```
Comment (abalakh) [inline robottelo/ui/locators/base.py]: explicit index isn't any better than text value in xpath. as fragile as text but also less readable. if there's nothing better to use like class or id, i'd rather prefer text over index. @oshtaier thoughts?

Comment (quarckster) [reply robottelo/ui/locators/base.py]: Here is the html code of the page.
```html
<div ng-app="RedhatAccessInsights">
    <!----><div ui-view="" class="wrapper in-prod"><div class="section">
    <section>
        <h3>
           General Information
        </h3>
        <p>
          For an overview of the Red Hat Insights service, please refer to the <a href="https://access.redhat.com/insights/info/" target="_blank">general information page</a>.
        </p>
        <h3>
           Getting Started
        </h3>
        <p>
          For help with configuring and enabling the Red Hat Insights service in Satellite 6, please refer to the <a href="https://access.redhat.com/insights/getting-started/satellite/6/" target="_blank">Getting Started Guide</a>.
        </p>
    </section>
</div>
</div>
    <div class="actionbar"><!----></div>
  </div>
```

Comment (oshtaier) [reply robottelo/ui/locators/base.py]: Yeah, that is not best style of locators. Sometimes, that questions the whole idea whether we need to test/address such UI sections at all

Comment (oshtaier) [reply robottelo/ui/locators/base.py]: but anyway 6.3 is not so important at that moment, so let's ignore such facts

**Comment by quarckster** [comment]
Tests results:
```
▶ pytest -v tests/foreman/rhai/test_rhai.py -k "test_navigation or test_rhai_manage_service or test_rhai_manage_insights_connection"
== test session starts ==
platform linux -- Python 3.6.5, pytest-3.6.1, py-1.5.4, pluggy-0.6.0 -- /home/dmisharo/insights_env/bin/python3
cachedir: .pytest_cache
metadata: {'Python': '3.6.5', 'Platform': 'Linux-4.12.14-lp150.12.4-default-x86_64-with-glibc2.3.4', 'Packages': {'pytest': '3.6.1', 'py': '1.5.4', 'pluggy': '0.6.0'}, 'Plugins': {'wait-for': '1.0.9', 'xdist': '1.22.2', 'services': '1.2.1', 'pudb': '0.6', 'mock': '1.6.3', 'metadata': '1.7.0', 'html': '1.19.0', 'forked': '0.2', 'cov': '2.5.1'}, 'JAVA_HOME': '/usr/lib64/jvm/jre'}
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/dmisharo/insights_env/src/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.2, services-1.2.1, pudb-0.6, mock-1.6.3, metadata-1.7.0, html-1.19.0, forked-0.2, cov-2.5.1
collecting 6 items                                                                                                                                                                                                                          2018-07-27 11:44:06 - conftest - DEBUG - BZ deselect is disabled in settings

collected 6 items / 3 deselected

tests/foreman/rhai/test_rhai.py::RHAITestCase::test_navigation PASSED [ 33%]
tests/foreman/rhai/test_rhai.py::RHAITestCase::test_rhai_manage_insights_connection PASSED [ 66%]
tests/foreman/rhai/test_rhai.py::RHAITestCase::test_rhai_manage_service PASSED [100%]

== 3 passed, 3 deselected in 113.29 seconds ==
```

**Pull request merged**